packages/nodes-base/scripts/build.mjs

```diff
@@ -0,0 +1,45 @@
+import path from 'path';
+import glob from 'fast-glob';
+import esbuild from 'esbuild';
+import {fileURLToPath} from 'url';
+
+/**
+ * Configuration
+ */
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const watchMode = process.argv[2] === '--watch';
+const rootDir = path.resolve(__dirname, '..');
+const distDir = path.resolve(rootDir, 'dist');
+
+/**
+ * Build
+ */
+
+const tsFiles = (await glob([path.resolve(rootDir, '{credentials,nodes,src}/**/*.ts')]))
+	.filter((file) => !file.endsWith('.d.ts'));
```

Comment:
1. Removed `src`
2. That's essentially what the glob does as well, and it runs only once. The overhead is `tsc`, unfortunately. The `build.mjs` script runs in a few hundred milliseconds.

Reply:
2. But after glob there is filter so double iteration? `walk` is intended to avoid double iteration as it includes `filter` as `test`. Let me know if I'm misunderstanding.